BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Giannini/ProgrammaticSliderViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+import AVFoundation
+
+class ProgrammaticSliderViewController: UIViewController {
+    let slider = UISlider()
+    var imageView : UIImageView!
+    var songPlayer: AVAudioPlayer?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.white
+        setupSlider()
+        setupBackground()
+        layoutConstraints()
+        DispatchQueue.global(qos: .userInitiated).async { self.playSong() }
+        }
+    func setupSlider() {
+        view.addSubview(slider)
+        slider.addTarget(self, action: #selector(self.changeSliderValue(_:)), for: .valueChanged)
+        slider.minimumValue = 0.0
+        slider.maximumValue = 1.0
+        slider.value = 1
+        slider.maximumTrackTintColor = UIColor.black
+        slider.thumbTintColor = UIColor.blue
+        slider.isContinuous = true
+    }
+    func layoutConstraints() {
+        slider.translatesAutoresizingMaskIntoConstraints = false
+        slider.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        slider.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -100).isActive = true
+        slider.widthAnchor.constraint(equalToConstant: 250).isActive = true
+        slider.heightAnchor.constraint(equalToConstant: 50).isActive = true
+    }
+    @objc func changeSliderValue(_ sender: UISlider) {
+        print("value is \(sender.value)");
```

Comment:
Avoid leaving `print` code in your commits.

Reply:
Print statements removed